README.md

```diff
@@ -271,7 +271,8 @@ const renderforest = new Renderforest({ signKey: '<signKey>', clientId: -1 })
 
 const payload = {
   projectId: 5000658,
-  quality: 1080
+  quality: 360,
+  watermark: 'https://somedomain.com/watermark.png' // optional argument
```

Comment:
in the whole repository, it's a convention to use example.com for a mock value of the url.